tests/tests_api/test_teacher_under_18.py

```diff
@@ -30,7 +32,9 @@ def test_teacher_under_18_retrieve(api_client):
     assert response.json() == {
         "personal_info": teacher_under_18.personal_info.id,
         "comment": teacher_under_18.comment,
-        "additional_skills_comment": teacher_under_18.additional_skills_comment,
-        "can_help_with_speaking_club": teacher_under_18.can_help_with_speaking_club,
+        "can_host_speaking_club": teacher_under_18.can_host_speaking_club,
         "status": teacher_under_18.status,
+        "status_since": teacher_under_18.status_since.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
I think some of that should work, either `str(teacher_under_18.status_since)` or `teacher_under_18.status_since.isoformat()`

Reply:
Nah...

First solution gives:
```
{'status_since': '2023-04-09T11:54:51.645249Z'} != {'status_since': '2023-04-09 11:54:51.645249+00:00'}
```

Second solution gives:
```
{'status_since': '2023-04-09T11:53:34.353149Z'} != {'status_since': '2023-04-09T11:53:34.353149+00:00'}
```

Reply:
That's weird behaviour in either python's datetime or drf.

I recommend to keep it as 
teacher_under_18.status_since.isoformat().replace("+00:00", "Z")

because in DRF datetime field source code there is:
```python
        if output_format.lower() == ISO_8601:
            value = value.isoformat()
            if value.endswith('+00:00'):
                value = value[:-6] + 'Z'
            return value
```
🤯

Reply:
I think it's because of mismatch of formats in JS and Python